hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java

```diff
@@ -128,13 +135,21 @@ public void checkSolvency(
             throw new InsufficientNetworkFeeException(INSUFFICIENT_TX_FEE, totalFee);
         }
         if (availableBalance < fees.networkFee()) {
-            throw new InsufficientNetworkFeeException(INSUFFICIENT_PAYER_BALANCE, totalFee);
+            if (ingestCheck) { // throw different exception for ingest
+                throw new InsufficientNetworkFeeException(INSUFFICIENT_PAYER_BALANCE, totalFee);
+            } else {
+                throw new InsufficientNetworkFeeException(INSUFFICIENT_ACCOUNT_BALANCE, totalFee);
+            }
```

Comment:
If these exceptions can be the same, then we can simplify the throw blocks a bit by extracting the conditional.
At the beginning of the method, we can set the response code
```java
        final ResponseCodeEnum insufficientFundsResponse;
        if (ingestCheck) { // use a different status for ingest
            insufficientFundsResponse = INSUFFICIENT_PAYER_BALANCE;
        } else {
            insufficientFundsResponse = INSUFFICIENT_ACCOUNT_BALANCE;
        }
```
```suggestion
            throw new InsufficientNetworkFeeException(insufficientFundsResponse, totalFee);
```

Reply:
Yea, thought this code would be removed later and just did quick copy/paste. Updated now.